src/decode-response/handle-rpc-error.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var abi = require("augur-abi");
+var errors = require("../errors/codes");
+
+var handleRPCError = function (method, returns, response) {
+  var i, len, responseNumber;
+  if (response) {
```

Comment:
Googling a bit reveals that the holy war between early-returns and single-point-of-exit rages on.  I try and be pragmatic about it.  I think I agree with you in this case though that an early return would be cleaner.